docs/how-to/what-is-vm-templating-and-how-do-I-use-it.md

```diff
@@ -47,7 +47,7 @@ and can be classified as potentially misunderstood behaviors rather than vulnera
 VM templating can be enabled by changing your Kata Containers config file (`/usr/share/defaults/kata-containers/configuration.toml`,
 overridden by `/etc/kata-containers/configuration.toml` if provided) such that:
 
-  - `qemu-lite` is specified in `hypervisor.qemu`->`path` section
+  - `qemu` version `v4.1.0` or above is specified in `hypervisor.qemu`->`path` section
```

Comment:
We've already merged changes which require qemu v5.0 or later to work at all, so the specific restriction to qemu v4.1.0 is redundant

Reply:
Oh, sorry, didn't spot that this is a backport branch.  I don't think we do have the v5.0 requirement in stable-2.2, so never mind.